docs/installation.rst

```diff
@@ -1,5 +1,170 @@
+
+::
+    subscription-manager repos --enable ansible-2.8-for-rhel-8-x86_64-rpms
+    yum install ansible
+    yum install ansible-freeipa
+
+
+The location where the ansible roles are installed:
+
+::
+    [root@server]# ls -1 /usr/share/ansible/roles/
+    ipaclient
+    ipareplica
+    ipaserver
+
+
+The location where the documentation is once installed:
+
+::
+    [root@server]# ls -1 /usr/share/doc/ansible-freeipa/
+    playbooks
+    README-client.md
+    README.md
+    README-replica.md
+    README-server.md
+    README-topology.md
+
+
+The location where the ansible playbooks are once installed:
+
+::
+    [root@server]# ls -1 /usr/share/doc/ansible-freeipa/playbooks/
+    install-client.yml
+    install-cluster.yml
+    install-replica.yml
+    install-server.yml
+    uninstall-client.yml
+    uninstall-cluster.yml
+    uninstall-replica.yml
+    uninstall-server.yml
+
+
+A sample playbook making use of the role to install the server:
+
+::
+    [root@server]# cat install_server.yaml
+    ---
+    - name: Playbook to configure IPA server
+      hosts: ipaserver
+      become: true
+
+      roles:
+      - role: ipaserver
+        state: present
+
+    [root@server]# ansible-playbook -i inventory install_server.yaml
+
+
+
+A sample playbook making use of the install a replica role:
+
+::
+    [root@server]# cat install_replica.yaml
+    ---
+    - name: Playbook to configure IPA replicas
+      hosts: ipareplicas
+      become: true
+
+      roles:
+      - role: ipareplica
+        state: present
+
+    [root@server]# ansible-playbook -i inventory install_replica.yaml
+
+
+===================
+Noggin Installation
+===================
+
+Clone the Fedora infrastructure ansible repo:
+
+::
+    [root@server]# git clone https://pagure.io/fedora-infra/ansible.git
+
+
+The ansible playbooks require a number of variables to be set and available:
+
+::
+    [root@server]# cat extravars.yml
+    # These variables will be used when env == 'production'
+    noggin_admin_password: "somepassword"                                      # IPA password for the user noggin
+    noggin_fernet_secret: "G8ObvrpEEwbjWUO9rU1qAkDQRafAFd39heVKYf6TZi8="       # Fernet Cryptography key
+    noggin_session_secret: "monkiesmonkiesmonkiesmonkiesmonkies!!!1111monkies" # Session Secret
+    noggin_github_secret: ""                                                   # Openshift github webhook, not needed
+    env: "production"
+    env_suffix: ""                                                             # Empty for production
+
+    # These variables will be used when env == 'dev' or set to any value other than 'production'
+    # noggin_stg_admin_password: ""
+    # noggin_stg_fernet_secret: ""
+    # noggin_stg_session_secret: ""
+    # noggin_stg_github_secret : "" # Openshift github webhook, not needed
+    # env: "staging"
+    # env_suffix: ".stg"
+
+    noggin_theme: ""                                                           # 
```

Comment:
@abompard I'm not sure what goes here.

Reply:
wait nm found it, https://github.com/fedora-infra/noggin/blob/dev/noggin.cfg.default#L13-L14

So "default" is enough.